### ISSUE TITLE
Who uses the sshtunnel?

### ISSUE THREAD

**Issue by pahaz** [pahaz/sshtunnel#181]
There are some popular repos and cases:
 - `airflow` (17k stars) https://github.com/apache/airflow/pull/3473
 - `redash` (16k stars) https://github.com/getredash/redash/pull/4797
 - `azure-cli` (2k stars) https://github.com/Azure/azure-cli/pull/1108
 - `OmniDB` (2k stars) https://github.com/OmniDB/OmniDB/pull/526
 - `Archery` (1.5k stars) https://github.com/hhyo/Archery/pull/736
 - `pgadmin4` (0.5k stars) https://github.com/postgres/pgadmin4/commit/b7fb01ab04e69fe558f0f2654f9940c549529959

**Comment by V0idk** [comment]
I suggest this project release a version every six months even if nothing updated，because some teams will estimate if project is active or outdated

**Comment by mdavis-xyz** [comment]
Releasing a new version just for the sake of it seems like a waste of time. Only release a new version if there's bug fixes or new features that users want.

**Comment by mdavis-xyz** [comment]
@pahaz what's the purpose of this issue? Do you want us to describe the use cases we have?

**Comment by pahaz** [comment]
Yes. It is a list of the most popular projects using the sshtunnel. And there is a place where you can find it.
